tests/validLogin.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
+  await page.goto('');
+  await page.locator('[data-test="email"]').fill('customer@practicesoftwaretesting.com');
+  await page.locator('[data-test="password"]').fill('welcome01');
+  await page.locator('[data-test="login-submit"]').click();
```

Comment:
Please consider this option in config file https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute in order to use getByTestId() method